test/ir_Panasonic_test.cpp

```diff
@@ -575,6 +575,15 @@ TEST(TestIRPanasonicAcClass, SetAndGetModel) {
   pana.setRaw(jkeState);
   EXPECT_EQ(kPanasonicJke, pana.getModel());
   EXPECT_STATE_EQ(jkeState, pana.getRaw(), kPanasonicAcBits);
+
+  // This state tickled a bug in getModel(). Should read as CKP.
+  uint8_t ckpState[27] = {0x02, 0x20, 0xE0, 0x04, 0x00, 0x00, 0x00, 0x06,
+      0x02, 0x20, 0xE0, 0x04, 0x00, 0x67, 0x2E, 0x80, 0xAF, 0x00, 0xC0,
+      0x6B, 0x98, 0x10, 0x00, 0x81, 0x64, 0x05, 0x87};
+  pana.setModel(kPanasonicDke);  // Make sure it isn't some how set to CKP
```

Comment:
Typo? Shouldn't in the comment be "somehow"?

Reply:
Fixed. Thanks for spotting it.